hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/Text.java

```diff
@@ -226,7 +226,13 @@ public void set(String string) {
    * Set to a utf8 byte array.
    */
```

Comment:
Please update JavaDocs to provide details on behaviors as written here.

Reply:
OK, I would.